core/dbt/graph/selector.py

```diff
@@ -25,8 +25,7 @@ def get_package_names(nodes):
 def alert_non_existence(raw_spec, nodes):
     if len(nodes) == 0:
         warn_or_error(
-            f"The selector '{str(raw_spec)}' does not match any nodes and will"
-            f" be ignored"
+            f"The selection criterion '{str(raw_spec)}' does not match any nodes"
```

Comment:
`flake8` just takes issue with this line being two characters too long:
```
core/dbt/graph/selector.py:28:80: E501 line too long (81 > 79 characters)
```
So perhaps:
```suggestion
            f"The selection criterion '{str(raw_spec)}' does not match"
            f" any nodes"
```